README.md

```diff
@@ -34,6 +34,49 @@ On your root component define the Split provider:
 </SplitProvider>
 ```
 
+### Performance
+Note that if your `SDK_CONFIG_OBJECT` is defined inside of a component it will create unnecessary work for `SplitProvider`,
+because the object will have a different identity each render (`previousConfig !== newConfig`).
+
+Instead define config outside of your component:
+```tsx
+const config = { ... };
+
+const Root = () => (
+  <SplitProvider config={config}>
+    <App />
+  </SplitProvider>
+)
+```
+Or if you need to configure dynamically, memoize the object:
+```tsx
+const MySplitProvider = ({ trafficType, children }) => {
+  const config = React.memo(() => ({
```

Comment:
`React.memo` expects a react component not a function, I think you meant `useMemo` which would make sense here.

Reply:
🤦‍♂